NitroxClient/MonoBehaviours/NitroxDebugManager.cs

```diff
@@ -84,7 +84,8 @@ public void ToggleDebugging()
             {
                 UWE.Utils.PopLockCursor();
                 HideDebuggers();
-            }
+                Debuggers.ForEach(debugger => debugger.ResetWindowPosition());
```

Comment:
Use `foreach` to because functional-style programming should not cause side-effects

Reply:
@Measurity In this case I beg to differ; `ForEach` has no return value (unlike `.Select` or other aggregation helpers) meaning it is really only useful when causing side-effects.

Reply:
Because the designers of the C# language think it was a bad design on their part to include `ForEach` which is why it is not on an `IEnumerable<>`.

https://docs.microsoft.com/en-us/archive/blogs/ericlippert/foreach-vs-foreach

Reply:
But assigning to a temporary or writing a (usually "long") LINQ expression within a `foreach ()` doesn't look that neat IMO, though in this case where no other transforms are used I agree.

Besides, I'm certain Nitrox has functional code _somewhere_ (outside of `.ForEach`) that causes side-effects - likely written by me. Convenience and readability dominate paradigm :man_shrugging: